### PR TITLE
05 interfaces procfs

### PR DESCRIPTION
**Pull request by DomninDmitry-dev**
The simple procfs module

### CODE REVIEW

File: 05-Interfaces/mymodule.c [hunk 1]
```diff
@@ -2,16 +2,66 @@
...
+	}
+
+	fstr = proc_create(CONV_FILE_NAME, 0666, dir, &myops);
+	if (fstr == NULL) {
+		pr_err("mymodule-procfs: error creating procfs entry 1\n");
+		remove_proc_entry(DIR_NAME, NULL);
+		return -ENOMEM;
+	}
+
+	flog = proc_create(LOG_FILE_NAME, 0666, dir, &myops);
```
Comment (an1kh) [inline 05-Interfaces/mymodule.c]: Should be read only

**Review by an1kh — CHANGES_REQUESTED** [review]
This will work, but not a good solution.
Better to have 2 separate file_operations, one RW for converter, and another RO for stats.
This will also eliminate the need to check paths every call.

File: 05-Interfaces/mymodule.c (hunk 1 above)
Comment (DomninDmitry-dev) [reply 05-Interfaces/mymodule.c]: ok

File: 05-Interfaces/mymodule.c [hunk 2]
```diff
@@ -12,43 +12,138 @@
...
 	if (fstr == NULL) {
-		pr_err("mymodule-procfs: error creating procfs entry 1\n");
+		pr_err("mymodule-procfs: error creating procfs file: %s\n",
+			CONV_FILE_NAME);
 		remove_proc_entry(DIR_NAME, NULL);
 		return -ENOMEM;
 	}
 
-	flog = proc_create(LOG_FILE_NAME, 0666, dir, &myops);
+	flog = proc_create(LOG_FILE_NAME, 0644, dir, &slog_ops);
```
Comment (an1kh) [inline 05-Interfaces/mymodule.c]: Read-only for everyone 0444

**Review by an1kh — APPROVED** [review]
Please fix access rights and then merge without review.

**Pull request merged**